hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/OMClientRequest.java

```diff
@@ -180,6 +181,17 @@ protected OMResponse createErrorOMResponse(OMResponse.Builder omResponse,
     return omResponse.build();
   }
 
+
+  /*
+   * This method sets the omRequest. This will method will be called when
```

Comment:
@bharatviswa504 can we eliminate this method as a requirement for implementors to call. Since the preExecute always returns an OM request, the caller can take the result and update omRequest.

It would be a good idea to minimize the work done by preExecute/validateAndUpdateCache and make as much common as possible.

Reply:
This is added so that it will help in Non-HA, to avoid the creation of OMClientRequest object multiple times. This is not mandatory for HA.

Reply:
And also now PreExecute will be implemented by requests when there is a need to change OMRequest. So modified the preExecute from the abstract method, and implemented it in OMClientRequest which is base class for all the requests(Write).

Reply:
Okay got it. Done.